<launch>
	
	<arg name="manager" default="robot_movement_controller_standalone_nodelet"/>

	<arg name="GoalZ" default="1.0" /> 
	<arg name="ZScale" default="0.95" /> 
	<arg name="XScale" default="0.9" />
	<arg name="GoalX" default="0.0" />
	
	<arg name="GoalInterval" default="0.01" />
	<arg name="DumperDivisor"    default="1" />
	<arg name="MaxRobotSpeed"    default="1" />
	
  	<node pkg="nodelet" type="nodelet" name="$(arg manager)"  args="manager" output="screen"/>
  	
  	<node pkg="nodelet" type="nodelet" name="realsense_robot_control"
	args="load realsense_robot_control/RobotMovementControlNodelet $(arg manager)

        ZScale $(arg ZScale)
        XScale $(arg XScale)
	GoalZ $(arg GoalZ)
	GoalX $(arg GoalX)
	
	DumperDivisor $(arg DumperDivisor)
        
        GoalInterval $(arg GoalInterval)
	MaxRobotSpeed $(arg MaxRobotSpeed)">
	
			<remap from="goal1" to="/person_follower/goal" />
			<remap from="goal2" to="/depth_follower/goal" />
	</node>

  
</launch> 
